### ISSUE TITLE
userId and secretKey options not used

### ISSUE THREAD

**Issue by plequang** [aerobatic/yoke-cli#3]
Hi (again, sorry),

When userId and secretKey options are set, they are actually not used.
This is because of an error in the naming of the properties, in yoke.js line 92
https://github.com/aerobatic/yoke-cli/blob/master/bin/yoke.js#L92

```
    if (program.requireCredentials === true && !(program.user && program.key)) {
```

should be
```
    if (program.requireCredentials === true && !(program.userId && program.secretKey)) {
```

**Comment by dvonlehman** [comment]
Great catch. The unattended mode clearly has not been thoroughly put through its paces yet. Are you setting up an automated CI build? This has been published to npm as 0.0.24.

**Comment by plequang** [comment]
Thanks, userId and secretKey are working now.
Yes, I've already set up a CI build using grunt-aerobatic on jenkins, and now I'm trying to migrate to yoke-cli.
